hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -1613,21 +1595,6 @@ private static int getFileIdLengthWithoutFileIndex(String fileId) {
     return fileId.endsWith("-0") ? fileId.length() - 2 : fileId.length();
   }
 
-  /**
-   * Create the timestamp for a clean operation on the metadata table.
-   */
-  public static String createCleanTimestamp(String timestamp) {
-    return timestamp + OperationSuffix.CLEAN.getSuffix();
-  }
-
-  public static String createRollbackTimestamp(String timestamp) {
-    return timestamp + OperationSuffix.ROLLBACK.getSuffix();
-  }
-
-  public static String createRestoreTimestamp(String timestamp) {
-    return timestamp + OperationSuffix.RESTORE.getSuffix();
-  }
```

Comment:
Since the suffices are no longer used, should those suffix enums (e.g., `OperationSuffix.RESTORE`) be removed?

Reply:
The async index still needs that.

Reply:
Finally we can remove all the suffices.

Reply:
Finally we also remove the suffix for async index delta commit.